splinterd/src/daemon.rs

```diff
@@ -940,25 +971,25 @@ impl From<ConnectError> for StartError {
 
 impl From<ConnectionError> for StartError {
     fn from(connection_error: ConnectionError) -> Self {
-        StartError::NetworkError(format!("Network Error: {:?}", connection_error))
+        StartError::NetworkError(format!("{:?}", connection_error))
```

Comment:
Not sure if it makes any difference, but this can be accomplished with `err.to_string()` as well.

Reply:
updated